### ISSUE TITLE
Resolving dependencies (((

### ISSUE THREAD

**Issue by Kota-Kun** [sferik/rails_admin#2619]
➜  bundle                                                                                                                                                                                                                                                 ruby-2.3.0 
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies.............................................
Bundler could not find compatible versions for gem "rack":
  In Gemfile:
    rails (< 5.1, >= 5.0.0.beta3) was resolved to 5.0.0.beta3, which depends on
      railties (= 5.0.0.beta3) was resolved to 5.0.0.beta3, which depends on
        actionpack (= 5.0.0.beta3) was resolved to 5.0.0.beta3, which depends on
          rack (~> 2.x)

    rails_admin was resolved to 0.8.1, which depends on
      rack-pjax (~> 0.7) was resolved to 0.7.0, which depends on
        rack (~> 1.3)

    devise was resolved to 4.0.0, which depends on
      warden (~> 1.2.3) was resolved to 1.2.6, which depends on
        rack (>= 1.0)

    devise was resolved to 4.0.0, which depends on
      warden (~> 1.2.3) was resolved to 1.2.6, which depends on
        rack (>= 1.0)

**Comment by spickermann** [comment]
I just tried to `bundle update` against the current master. It seems like there is still a incompatibility on `rack` left to fix.

```
$ bundle update
Fetching https://github.com/sferik/rails_admin.git
Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Fetching dependency metadata from https://rubygems.org/
Resolving dependencies.................................
Bundler could not find compatible versions for gem "rack":
  In Gemfile:
    rails (>= 5.0.0.rc1) was resolved to 5.0.0.rc1, which depends on
      actioncable (= 5.0.0.rc1) was resolved to 5.0.0.rc1, which depends on
        actionpack (= 5.0.0.rc1) was resolved to 5.0.0.rc1, which depends on
          rack (~> 2.x)

    rails_admin was resolved to 0.8.1, which depends on
      rack-pjax (~> 0.7) was resolved to 0.7.0, which depends on
        rack (~> 1.3)
```

**Comment by luislew** [comment]
Same issue: `rack-pjax` v0.7 depends on sub-2 `rack`. Updating `rack-pjax` to `1.0.0` would fix this dependency error.

**Comment by mshibuya** [comment]
Resolved in 1.0.0.rc.
